tests/tests/src/functors_one_arg_at_a_time.res

```diff
@@ -0,0 +1,17 @@
+module Make = (T: {}, Q: {}) => {
+  module Eq = (E: {}, A: {}) => {}
+}
+
+module M = Make((), ())
+
+module EQ = M.Eq((), ())
+
+module MF: {
+  module F: (X: {}, Y: {}) => {}
```

Comment:
This is somehow still emitting a function of 2 args, when the `MF` is annotated with a module type.